spec/requests/vendors_requests_spec.rb

```diff
@@ -123,6 +123,20 @@
         get vendor_path(id: create(:vendor, organization: organization))
         expect(response).to be_successful
       end
+
+      it "displays purchases in reverse chronological order by issued_at date" do
+        vendor = create(:vendor, organization: organization)
+
+        # Create purchases with different issued_at dates
+        old_purchase = create(:purchase, vendor: vendor, issued_at: 1.week.ago, organization: organization)
+        new_purchase = create(:purchase, vendor: vendor, issued_at: 1.day.ago, organization: organization)
+        middle_purchase = create(:purchase, vendor: vendor, issued_at: 3.days.ago, organization: organization)
+
+        get vendor_path(vendor)
+
+        expect(vendor.reload.purchases.order(issued_at: :desc).to_a).to eq([new_purchase, middle_purchase, old_purchase])
```

Comment:
This isn't actually testing the view - you're just testing the database here. You need to inspect the response (i.e. `response.body`) to see that it contains the purchase dates in the expected order.